template/App.tsx

```diff
@@ -56,43 +56,56 @@ function Section({children, title}: SectionProps): React.JSX.Element {
 }
 
 function App(): React.JSX.Element {
+  return (
+    <SafeAreaProvider>
+      <MainContent />
+    </SafeAreaProvider>
+  );
+}
+
+function MainContent(): React.JSX.Element {
+  const insets = useSafeAreaInsets();
+
   const isDarkMode = useColorScheme() === 'dark';
 
   const backgroundStyle = {
     backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+    bottomPadding: insets.bottom,
   };
 
+  const contentContainerStyle = { paddingBottom: insets.bottom };
```

Comment:
need to add extra padding so when scrolling to the bottom, the content shows above the nav bar without overlap.